script/script.js

```diff
@@ -0,0 +1,326 @@
+      xMoves.includes(winningArrays[i][1].toString()) &&
+      xMoves.includes(winningArrays[i][2].toString())
+    ) {
+      //identify which were the winning cells
+      let winningCell1 = winningArrays[i][0];
+      let winningCell2 = winningArrays[i][1];
+      let winningCell3 = winningArrays[i][2];
+      //change color of winning cells
+      cellKey[winningCell1].style.backgroundColor = "red";
+      cellKey[winningCell2].style.backgroundColor = "red";
+      cellKey[winningCell3].style.backgroundColor = "red";
+      //message if player won in less than a minute
+      if (minutes.innerHTML == 00) {
+        playerTurn.textContent = `Congratulations, ${player1.value}, you won!!! It took you ${seconds.innerHTML} seconds!`;
+
+        //message if player won in over a minute
+      } else if (minutes.innerHTML !== 0) {
+        playerTurn.textContent = `Congratulations, ${player1.value}, you won!!! It took you ${minutes.innerHTML} minutes and ${seconds.innerHTML} seconds!`;
+      }
+      //stop the timer and show reset button
+      clearInterval(timeElapsed);
+      credit.style.display = "block";
+      reset.style.display = "block";
+    }
+    //check to see if array of player O's moves has a winning array by looping through
+    else if (
+      //check to see if array of player O's moves has a winning array by looping through each sub-array
+      oMoves.includes(winningArrays[i][0].toString()) &&
+      oMoves.includes(winningArrays[i][1].toString()) &&
+      oMoves.includes(winningArrays[i][2].toString())
+    ) {
+      //identify winning cells
+      let winningCell1 = winningArrays[i][0];
+      let winningCell2 = winningArrays[i][1];
+      let winningCell3 = winningArrays[i][2];
+      //change color of winning cells
+      cellKey[winningCell1].style.backgroundColor = "red";
+      cellKey[winningCell2].style.backgroundColor = "red";
+      cellKey[winningCell3].style.backgroundColor = "red";
+      //adjust message if player won in less than a minute, give winning message, stop timer
+      if (minutes.innerHTML == 00) {
+        playerTurn.textContent = `Congratulations, ${player2.value}, you won!!! It took you ${seconds.innerHTML} seconds!`;
+      }
+      //adjust message if player won in more than a minute, give winning message, stop timer
+      else if (minutes.innerHTML !== 0) {
+        playerTurn.textContent = `Congratulations, ${player2.value}, you won!!! It took you ${minutes.innerHTML} minutes and ${seconds.innerHTML} seconds!`;
+      }
+      //stop the timer and show reset button
+      clearInterval(timeElapsed);
+      credit.style.display = "block";
+      reset.style.display = "block";
+    }
+  }
+  //if no winning combos in either array & 9 cells are filled, it's a tie:
+  if (totalMoves.length === 9) {
+    playerTurn.textContent = "No winner :(";
+    clearInterval(timeElapsed);
+    credit.style.display = "block";
+    reset.style.display = "block";
+  }
```

Comment:
Per my comment above about wins, this is a really great and concise way to check over cells for a draw. Good use of array methods!